### ISSUE TITLE
Outgoing Webhook - USER.SUBMIT_FORM_DATA

### ISSUE THREAD

**Issue by Salzy** [circuit/circuit-sdk#63]
Hey @rogeru 

I'm not quite sure if it belongs here but i do have a feature request about the outgoing webhook _**USER.SUBMIT_FORM_DATA**_ response. 

Currently, whenever I submit a form, I get the following response (i.e. basically Trivia-example):
```
{ deliveryId: '45913582-2b2f-438c-8009-52ac9bbe70da',
  type: 'USER.SUBMIT_FORM_DATA',
  webhookId: '5b61389e-1f6e-43f9-83bf-af41217d5abc',
  submitFormData: 
   { itemId: 'ddfd5e9e-34cc-4c81-93c2-8097d7e6a4b5',
     formId: 'trivia',
     submitterId: 'f9d448f9-b887-4d2c-aff3-264b396c6da3',
     data: [ [Object], [Object] ] } }
```

Whenever I want to find out the conversation Id, I basically have to acquire the item information with **_itemId_**, which is an additional step.
Would it therefore be possible to add the corresponding **_convId_** in **_submitFormData_** response directly?

Thanks in advance.

**Comment by rogeru** [comment]
@Salzy yes that would simplify the apps which need the conversation ID. We can add this, but not sure on the timeframe.
